soc/nordic/nrf54h/Kconfig

```diff
@@ -67,3 +69,36 @@ config SOC_NRF54H20_CPUFLPR
 	depends on RISCV_CORE_NORDIC_VPR
 
 rsource "gpd/Kconfig"
+
+config SOC_NRF54H20_GLOBAL_HSFLL_RESTRICT_MIN_FREQ
+	bool "Restrict minimum global HSFLL clock frequency"
```

Comment:
The name is unfortunate as there is three global hsflls, 120, 121 and 200, I'd recommend renaming the GLOBAL_HSFLL to HSFLL120 throughout. I know it creates a detail level that most wouldn't want to be bothered with, but it will just make things confusing long term when it comes to dealing with the other global hsflls